docs/learn/learn-guides-polkadot-opengov.md

```diff
@@ -65,3 +65,20 @@ within the track, and a
 have been met. Failing to submit the decision deposit within a
 {{ polkadot: <RPC network="polkadot" path="const.referenda.undecidingTimeout" defaultValue={201600} filter="blocksToDays"/> :polkadot }}{{ kusama: <RPC network="kusama" path="const.referenda.undecidingTimeout" defaultValue={201600} filter="blocksToDays"/> :kusama }}-day
 period will lead to a referendum timeout.
+
+## Request Submission and Decision Deposit Back
+
+If you submitted a proposal and a referendum for such proposal has ended, you can claim your
+submission and decision deposits back by issuing the `refundSubmissionDeposit` and the
+`refundDecisionDeposit` extrinsics, respectively. Those functions will succeed if the proposal is
+completed as passing or failing.
+
+Users can not refund their deposit while the proposal is Ongoing. Similarly, users cannot refund
+their deposit if the proposal has TimedOut (see timeout conditions). This behavior exists so that
```

Comment:
Perhaps, link to the section with timeout conditions explained? or give an example of a track referenda timeout here.

```suggestion
their deposit if the proposal has `TimedOut` (see timeout conditions). This behavior exists so that
```